pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1456,6 +1456,16 @@ public class ServiceConfiguration implements PulsarConfiguration {
     )
     private String saslJaasServerSectionName = SaslConstants.JAAS_DEFAULT_BROKER_SECTION_NAME;
 
+    @FieldContext(
+            category = CATEGORY_SASL_AUTH,
+            doc = "Configure the secret to be used to SaslRoleTokenSigner\n"
+                    + "The secret can be specified like:\n"
+                    + "saslJaasServerRoleTokenSignerSecret=file:///my/saslRoleTokenSignerSecret.key\n"
+                    + "If saslJaasServerRoleTokenSignerSecret is empty, will use Default value "
+                    + "`SaslConstants.JAAS_DEFAULT_ROLE_TOKEN_SIGNER_SECRET`."
+    )
+    private String saslJaasServerRoleTokenSignerSecret;
```

Comment:
Similarly, we can have it default to the following:
```suggestion
    private String saslJaasServerRoleTokenSignerSecret = SaslConstants.JAAS_DEFAULT_ROLE_TOKEN_SIGNER_SECRET;
```

Reply:
There won't be a default here because a file path is required.